### PR TITLE
Country Filter: United States by Default

### PR DESCRIPTION
**Pull request by aramis-eb**
Country Filter: United States by Default

### CODE REVIEW

File: static/js/filterManager.js [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
+    }
+    
     function reloadLastFilters() {
         try {
             const filters = ["country", "online", "language", "format", "category"];
-    
             filters.forEach(function(filter) {
                 const lastValue = querySt(filter);
-                if(lastValue) document.getElementById(filter).value = lastValue;
+                if(lastValue != undefined){
```
Comment (julianrofrano-eb) [inline static/js/filterManager.js:16]: maybe you can replace this with if(lastValue)

File: static/js/filterManager.js [hunk 2]
```diff
@@ -25,7 +38,11 @@ const FilterManager = (function() {
         const div = document.getElementById(divId);
         const input = document.getElementById(inputId);
         if(value == 'on') input.value = '';
+        const visibility = div.style.visibility
         div.style.visibility = value == 'on' ? 'hidden' : 'visible';
+        if(div.style.visibility != visibility && div.style.visibility== 'visible' && visibility){
```
Comment (julianrofrano-eb) [inline static/js/filterManager.js:43]: please use Strict Equality Comparison (!== or ===)

Comment (aramis-eb) [reply static/js/filterManager.js]: Resolved!

File: static/js/filterManager.js (hunk 1 above)
Comment (aramis-eb) [reply static/js/filterManager.js:16]: We use "" when searching for all events. So we compare with undefined in this case

**Pull request merged**